rules/windows/process_creation/proc_creation_win_susp_lolbin_non_c_drive.yml

```diff
@@ -0,0 +1,32 @@
+title: Wscript Execution from Non C Drive
+id: 5b80cf53-3a46-4adc-960b-05ec19348d74
+status: experimental
+description: Detects Wscript or Cscript executing from a drive other than C. This has been observed with Qakbot executing from within a mounted ISO file.
+author: Aaron Herman
+date: 2022/10/01
+references:
+    - https://github.com/pr0xylife/Qakbot/blob/main/Qakbot_BB_30.09.2022.txt
+    - https://app.any.run/tasks/4985c746-601e-401a-9ccf-ae350ac2e887/
+tags:
+    - attack.execution
+    - attack.t1059
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_lolbin:
+        Image|endswith:
+            - '\wscript.exe'
+            - '\cscript.exe'
+    selection_exetensions:
+        CommandLine|contains: '.js'
+    selection_drive_path:
+        CommandLine|contains: ':\'
+    filter_drive_path:
+        CommandLine|contains: 'C:\'
```

Comment:
Filtering out the `C:\` would also exclude the process itself because the command line would most likely contain `C:\Windows\System32\WScript.exe`.

I think a maybe better way would be to focus on the name of the drive after the binary so for example 

`C:\Windows\System32\WScript.exe D:\` and we exclude `C:\Windows\System32\WScript.exe C:\`

Reply:
What's the best way to do this in Sigma? My understanding is that regex isn't supported for anything other than translations for Elasticsearch.

Reply:
C:\ is at the start of the CommandLine so should not have a space before  `' C:\'`

Reply:
True, but then we need to add a variation of quotes because it can be in 3 forms

- Double quotes: "C:\Windows\System32\WScript.exe" "D:\......"
- No quotes: "C:\Windows\System32\WScript.exe" D:\......
- Single quotes: "C:\Windows\System32\WScript.exe" 'D:\

So perhaps it becomes

```yml
CommandLine|contains:
  - ' C:\'
  - " 'C:\"
  - ' "C:\'
```

I know the single quote is rare but, I think that is the cleanest solution. Maybe? 

cc: @frack113 @Neo23x0